File: backend/src/main/kotlin/ch/boosters/backend/data/syncConfig/SyncConfig.kt
package ch.boosters.backend.data.syncConfig

import ch.boosters.data.tables.pojos.EventsTable
import ch.boosters.data.tables.pojos.SportsTable
import ch.boosters.data.tables.pojos.TeamsTable
import java.util.UUID

data class SyncConfig (
    val id: UUID?,
    val name: String,
    val teams: List<TeamsTable>,
    val sports: List<SportsTable>,
    val events: List<EventsTable>
)

File: frontend/src/services/syncConfig/entities/syncConfig.ts
import { Team } from "../../team/entities/team.ts";
import {Sport} from "../../sport/entities/sport.ts";
import {SportEvent} from "../../event/entities/event.ts";

export interface SyncConfig {
    id: string;
    name: string;
    sports: Sport[];
    teams: Team[];
    events: SportEvent[];
}
